hw/ip/otbn/dv/uvm/otbn_sim_cfg.hjson

```diff
@@ -53,20 +55,34 @@ name:
   uvm_test: otbn_base_test
   uvm_test_seq: otbn_base_vseq
 
-  // The directory where ELF binaries should be found.
-  //
-  // TODO: This needs deciding upon. Eventually, it might be nice to allow this
-  //       to be overridden on the command line ("I've got a bunch of ELF files
-  //       that stress my new feature; please run the test suite with them")
+  // A build mode that sets up the toolchain. Run meson via
+  // meson_init.sh to generate the .env file that describes our toolchain.
+  build_modes: [
+    {
+      name: otbn_build_env_mode
+      pre_build_cmds: ["cd {proj_root} && BUILD_ROOT={build_dir} ./meson_init.sh"]
+    }
+  ]
+
+  // The default place to find and build ELF binaries. If you have
+  // some pre-built binaries, you can run the simulation with them by using
+  // --run-opts to override the otbn_elf_dir plusarg in run_opts.
   otbn_elf_dir: "{run_dir}/build-out"
```

Comment:
Can this be:
```suggestion
  otbn_elf_dir: "{run_dir}"
```

`build-out` is an artifact of meson, which is perhaps not required to be added when running your script?

Reply:
Well, I think we should probably put the binaries in a subdirectory, not straight in `{run_dir}`. I didn't actually change the name from your code, but I agree that it's not a great one. Let's change it to `{run_dir}/otbn-binaries`, which is probably a bit clearer.

Reply:
Sure, that works.